components/resource_control/src/resource_limiter.rs

```diff
@@ -94,10 +102,12 @@ impl QuotaLimiter {
         self.limiter.set_speed_limit(limit);
     }
 
-    fn get_statistics(&self) -> (u64, u64) {
+    pub(crate) fn get_statistics(&self) -> (u64, u64, u64, u64) {
```

Comment:
not need to be pub, should depend on ResourceLimiter::get_limit_statistics instead